salt/engines/__init__.py

```diff
@@ -43,7 +43,7 @@ def start_engines(opts, proc_mgr):
 
     for engine in engines_opt:
         if isinstance(engine, dict):
-            engine, engine_opts = engine.items()[0]
+            engine, engine_opts = list(engine.items())[0]
```

Comment:
you are right @skizunov you need to keep the .items, since that will return a type with iter will correctly translate so that it returns two items